### ISSUE TITLE
nvda behaves strangely with multi line edit fields in internet explorer 8

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3547]
**Reported by fatma.mehanna on 2013-09-25 01:53**
hi,
NVDA behaves in a strange way when i type in multi line edit fields in internet explorer 8.
steps to reproduce:
1 type 2 or 3 paragraphs.
2 try to read what you typed.
resulsts:
you will find out that nvda reads the first paragraph fine. when you arrow down to read the second paragraph, it reports the words of the first one.
 if you want to be sure of the words of the paragraph you are reading, which should be the second one, press home and try to read the words one by one using ctrl+left/right arrow.
 you will find out that you are reading the words of the second paragraph.
try to arrow down to read the third paragraph, you will find out that nvda reads the words of the second one, etc.
some times it doesn't read the whole paragraph when you go line by line.
 it reads part of it.
when i type in the status update multi line edit field in m.facebook.com webpage,
it doesn't read the first paragraph and nvda says blank. when i go down 2 or 3 lines and go to the top again, nvda will read part of the first line.
when i go to browse mode and read what i typed, it reads perfect.
i use nvda Version: next-9786,c74811f

**Comment by leonardder** [comment]
This seems to be IE 8 specific. Furthermore, we can't contact the original author. Closing as won't fix.

**Comment by leonardder** [comment]
H
